apis/core/bootstrap/shapes/dimension.ts

```diff
@@ -46,9 +46,9 @@ ${shape('dimension/metadata')} {
     ]
   .
 
-  ${qudt.NominalScale} ${rdfs.label} "Nominal Scale"@en .
-  ${qudt.OrdinalScale} ${rdfs.label} "Ordinal Scale"@en .
-  ${qudt.IntervalScale} ${rdfs.label} "Interval Scale"@en .
-  ${qudt.RatioScale} ${rdfs.label} "Ratio Scale"@en .
+  ${qudt.NominalScale} ${rdfs.label} "Nominal"@en .
+  ${qudt.OrdinalScale} ${rdfs.label} "Ordinal"@en .
+  ${qudt.IntervalScale} ${rdfs.label} "Interval"@en .
+  ${qudt.RatioScale} ${rdfs.label} "Ratio"@en .
```

Comment:
@l00mi with only four, the choices are displayed as radio buttons. the repeated `Scale` makes it a little long